### PR TITLE
port toranj test35 child_timeout_large_data_poll

### PR DESCRIPTION
**Pull request by priyankaCh22**

### CODE REVIEW

File: silk/tests/openthread/ot_test_child_timeout_large_data_poll.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        self.child.set_sleep_poll_interval(str(5 * CHILD_TIMEOUT * 1000))
+
+        # Now join the child as sleepy-end-device to the thread network
+        self.child.join(self.network_data, "sleepy-end-device")
+        self.wait_for_completion(self.device_list)
+
+    @testcase.test_method_decorator
+    def test03_verify_child_sends_a_data_poll(self):
+        poll_count_before = int(self.child.get(wpan.WPAN_NCP_COUNTER_TX_PKT_DATA_POLL), 0)
+        time.sleep(1.375)
```
Comment (qilinkang) [inline silk/tests/openthread/ot_test_child_timeout_large_data_poll.py]: time.sleep(TIMEOUT * 1.1) 
Ensure testing data poll from child after Timeout setting.

Comment (priyankaCh22) [reply silk/tests/openthread/ot_test_child_timeout_large_data_poll.py]: Ack

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by tzm41 — APPROVED** [review]
LGTM

**Review by jwhui — APPROVED** [review]

**Pull request merged**